src/layers/merge_test.ts

```diff
@@ -376,3 +379,139 @@ describeMathCPUAndGPU('Concatenate Layer: Tensor', () => {
     });
   }
 });
+
+describeMathCPU('Deserialize Merge Layers', () => {
+  it('Model with Add Layer', () => {
+    // The following model config JSON can be obtained with Python code:
+    // ```python
+    // import keras
+    //
+    // input1 = keras.Input(shape=[4])
+    // input2 = keras.Input(shape=[4])
+    //
+    // output = keras.layers.add([input1, input2])
+    // model = keras.Model([input1, input2], output)
+    //
+    // print(model)
```

Comment:
Is this line intentional?  Does it provide value over the call to print(model_json)?

Reply:
same q in concat version below